### DISCUSSION TITLE
Using the newer MMI2 ports?

### DISCUSSION THREAD

**Discussion by abmantis** [raomin/ESPAltherma#241] [Q&A]
The MMI2 control board has a CAN port, and also two UART interface ports. Could any of these be used to read/write data?

![image](https://user-images.githubusercontent.com/974569/216739996-fbaaa35e-1c53-4cc0-9e15-0aec3972ded3.png)

**Comment by javimurcia** [comment]
If i'm not mistaken, the X9 and X5 ports are used by the new wlan adapter BRP069A71, i don't think they speak the same "language" as the X10a

**Comment by Tardymo** [comment]
Yes, X9 and X5 is for WLAN adapter, and X3 is where Bizone kit connects. However WLAN UART might be interesting as it would also allow to control the heat pump not just read values from it. Most likely completely different protocol.

**Reply by abmantis** [reply]
But the wlan adaptar goes in the sdcard port (X10). Is it also connected to X5 and X9? 
Is the bizone kit protocol known?

**Reply by abmantis** [reply]
Ah, just found out that there are some wlan adapter modules that connect to X5 and X9

**Reply by javimurcia** [reply]
That's correct, newer daikin models doesn't have the option for the traditional p1p2 wlan/ethernet gateway, instead they have the option for the sd-card wlan adapter, and an external wlan adapter using X5 and X9 ports, which i suppose for areas with poor wifi reception where the sd-card doesn't cut it, because as far as the documentation goes, both have the same capabilities.

**Reply by myLens** [reply]
This thread helped me finally locate the X3 connector. I did not expect it to be on THAT PCB as there are tons of connectors on the two mainboards (J101 even matching) but with no LON-transceiver anywhere around :-D 
So my BiZone kit is finally working and if I have time i will try to reverse engineer the LON frames to control and read pumps, actuators and sensors.
